platform/ui-next/src/components/Onboarding/Onboarding.tsx

```diff
@@ -7,15 +7,18 @@ import './Onboarding.css';
 
 import { hasTourBeenShown, markTourAsShown, defaultShowHandler, middleware } from './utilities';
 
-const Onboarding = () => {
-  const Shepherd = useShepherd();
-  const location = useLocation();
-  const tours = window.config.tours as Array<{
+const Onboarding = ({
+  tours,
```

Comment:
Could you assign an empty array by default so it doesn't have to have the tours assigned?  That should be allowed to work to define no tours.